draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -279,6 +281,12 @@ Implementers should be aware that using the same client attestation across multi
 
 The guidance provided by {{RFC7519}} and {{RFC8725}} applies.
 
+Replay attack prevention MUST be considered by the protocols using this client authentication method. Possible mechanisms comprise:
+
+- The client uses "jti" (JWT ID) claims for the Client Attestation PoP JWT and the authorization server maintains a list of used (seen) "jti" values for the time of which the JWT would be considered valid based on the applicable "exp" claim. If any Client Attestation PoP JWT would be replayed, the authorization server would recognize the "jti" and respond with an authentication error.
+- The authorization server provides a nonce for the particular transaction and the client uses it for the "nonce" claim in the Client Attestation PoP JWT. The authorization server validates that the nonce matches for the transaction. The authorization server MUST ensure that the nonce provides sufficient entropy.
+- The authorization server may expect the usage of a nonce in the Client Attestation PoP JWT, but instead of providing the nonce explicitly, the client may implicitly reuse an existing artefact, e.g. the authorization code. The authorization server MUST ensure that the nonce provides sufficient entropy.
```

Comment:
here we don't say who's the issuer of the existing artefact and from where and when it was taken.

I understand the meaning but for editorial sake we should say something more explicit here

Reply:
done in new PR